src/rendition-mixin.js

```diff
@@ -31,7 +31,7 @@ const enableFunction = (loader, playlistUri, changePlaylistFn, enable) => {
     // Ensure the outside world knows about our changes
     changePlaylistFn();
   }
-
+  loader.trigger('hlsrenditiondisabled');
```

Comment:
This should be moved into the if block just above.
```
rep.enabled(false);
rep.enabled(false);
```
will cause two triggers, but I don't think it makes sense to trigger the event twice when its not switching from enabled/disabled